trade-finance-manager-ui/server/controllers/utilisation-reports/helpers/reconciliation-for-report-helper.ts

```diff
@@ -251,6 +304,11 @@ export const mapFeeRecordPaymentGroupsToPaymentDetailsViewModel = (feeRecordPaym
           paymentIdToDateReceivedDataSortValueMap[payment.id],
         ),
         feeRecords: mappedFeeRecords,
+        reconciledBy: getFormattedReconciledByUser(reconciledByUser),
+        dateReconciled: {
+          formattedDateReconciled: getFormattedDateReconciled(dateReconciled),
+          dataSortValue: paymentIdToDateReconciledDataSortValueMap[payment.id],
+        },
```

Comment:
Not for this PR, it would be nice to split this helper file up as there's a lot going on and I imagine will only get bigger as we add more functions